apps/docs/content/guides/platform/compute-add-ons.mdx

```diff
@@ -9,14 +9,14 @@ Every project on the Supabase Platform comes with its own dedicated Postgres ins
 | Plan   | Hourly Price USD | Monthly Price USD | CPU                     | Memory | Connections: Direct | Connections: Pooler | Recommended Maximum DB Size |
 | ------ | ---------------- | ----------------- | ----------------------- | ------ | ------------------- | ------------------- | --------------------------- |
 | Micro  | $0.01344         | ~$10              | 2-core ARM (shared)     | 1 GB   | 60                  | 200                 | 10GB                        |
-| Small  | $0.0206          | ~$15              | 2-core ARM (shared)     | 2 GB   | 90                  | 200                 | 50GB                        |
-| Medium | $0.0822          | ~$60              | 2-core ARM (shared)     | 4 GB   | 120                 | 200                 | 100GB                       |
-| Large  | $0.1517          | ~$110             | 2-core ARM (dedicated)  | 8 GB   | 160                 | 300                 | 200GB                       |
-| XL     | $0.2877          | ~$210             | 4-core ARM (dedicated)  | 16 GB  | 240                 | 700                 | 500GB                       |
-| 2XL    | $0.562           | ~$410             | 8-core ARM (dedicated)  | 32 GB  | 380                 | 1500                | 1TB                         |
-| 4XL    | $1.32            | ~$960             | 16-core ARM (dedicated) | 64 GB  | 480                 | 3000                | 2TB                         |
-| 8XL    | $2.562           | ~$1,870           | 32-core ARM (dedicated) | 128 GB | 490                 | 6000                | 4TB                         |
-| 12XL   | $3.836           | ~$2,800           | 48-core ARM (dedicated) | 192 GB | 500                 | 9000                | 6TB                         |
+| Small  | $0.0206          | ~$15              | 2-core ARM (shared)     | 2 GB   | 90                  | 400                 | 50GB                        |
```

Comment:
should we add nano to this table?

Reply:
there's another pr for nano: https://github.com/supabase/supabase/pull/22311